CONTRIBUTING.md

```diff
@@ -4,15 +4,15 @@ Please familiarize yourself with the SPDX License List and its supporting docume
 
 * [Explanation of the fields contained on the list](DOCS/license-fields.md)
 
-* [SPDX License List Matching Guidelines](https://spdx.org/spdx-license-list/matching-guidelines) provides guidelines to be used for the purposes of matching licenses and license exceptions against those included on the SPDX License List.
+* [SPDX License List Matching Guidelines]([https://spdx.org/spdx-license-list/matching-guidelines](https://spdx.github.io/spdx-spec/v2.3/license-matching-guidelines-and-templates/)) provides guidelines to be used for the purposes of matching licenses and license exceptions against those included on the SPDX License List.
```

Comment:
You added link to link. That does not have sense. The old link is dead so it should be updated - that is correct. But if we keep the version we will have to update it every second year. You can use https://spdx.github.io/spdx-spec/latest/license-matching-guidelines-and-templates/  which redirect to latest tag. Right now to v2.3. That should work in future without need for a change.

Reply:
fixed now